core/cafe_api.go

```diff
@@ -24,12 +24,14 @@ import (
 	"github.com/textileio/textile-go/pb"
 )
 
+// CafeInfo details info about this cafe
 type CafeInfo struct {
 	Peer     string `json:"peer"`
 	Address  string `json:"address"`
 	API      string `json:"api"`
 	Protocol string `json:"protocol"`
 	Node     string `json:"node"`
+	URL      string `json:"url"`
```

Comment:
The URL needed for registration is provided here. Useful when doing the gateway cafe discovery thing.